packages/web/components/input/limit-input.tsx

```diff
@@ -124,13 +124,81 @@ export const LimitInput: FC<LimitInputProps> = ({
     }
   }, [tab, type]);
 
-  const setFiatAmountSafe = useCallback(
-    (value?: string) => {
-      if (!value) {
-        return setFiatAmount("");
+  // const setFiatAmountSafe = useCallback(
```

Comment:
The amalgamation is a lot cleaner, I think we can delete the commented code here